go/service/teams.go

```diff
@@ -158,6 +158,9 @@ func (h *TeamsHandler) TeamRemoveMember(ctx context.Context, arg keybase1.TeamRe
 	if len(arg.Email) > 0 {
 		h.G().Log.CDebugf(ctx, "TeamRemoveMember: received email address, using CancelEmailInvite for %q in team %q", arg.Email, arg.Name)
 		return teams.CancelEmailInvite(ctx, h.G().ExternalG(), arg.Name, arg.Email)
+	} else if len(arg.InviteID) > 0 {
+		h.G().Log.CDebugf(ctx, "TeamRemoveMember: received InviteID, using CancelInviteByID for %q in team %q", arg.InviteID, arg.Name)
+		return teams.CancelInviteByID(ctx, h.G().ExternalG(), arg.Name, arg.InviteID)
```

Comment:
not exposed to CLI right now, should it be?

Reply:
Yeah, probably. Make a ticket for next sprint?